components/Cards/index.js

```diff
@@ -6,14 +6,102 @@
 // This won't be as easy as just iterating over an array though.
 // Create a function that will programmatically create the following DOM component:
 //
-// <div class="card">
-//   <div class="headline">{Headline of article}</div>
-//   <div class="author">
-//     <div class="img-container">
-//       <img src={url of authors image} />
-//     </div>
-//     <span>By {authors name}</span>
-//   </div>
-// </div>
+/* 
+<div class="card">
+  <div class="headline">{Headline of article}</div>
+  <div class="author">
+    <div class="img-container">
+      <img src={url of authors image} />
+    </div> 
+    <span>By {authors name}</span>
+  </div>
+</div> 
+*/
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+// GET data from provided server
+let articleAPI = axios.get('https://lambda-times-backend.herokuapp.com/articles');
+
+// SELECT the class I need to append everything to
+let cardContainer = document.querySelector('.cards-container');
+
+
+// a function that will programmatically create the above DOM component:
+function lambCards(obj){
+    // create elements
+    let parentDiv = document.createElement('div');
+    let headlineDiv = document.createElement('div');
+    let authorDiv = document.createElement('div');
+    let imgDiv = document.createElement('div');
+    let img = document.createElement('img');
+    let authNameSpan = document.createElement('span');
+    
+    
+    //add class list
+    parentDiv.classList.add('card');
+    headlineDiv.classList.add('headline');
+    authorDiv.classList.add('author');
+    imgDiv.classList.add('img-container');
+    
+    //structure / append
+    parentDiv.appendChild(headlineDiv);
+    parentDiv.appendChild(authorDiv);
+            //**
+    authorDiv.appendChild(imgDiv);
+    authorDiv.appendChild(authNameSpan);
+          //**
+    authorDiv.appendChild(authNameSpan);   
+    imgDiv.appendChild(img);
+    
+    //text content and img src
+    headlineDiv.textContent = `${obj.headline}`;
+
+    img.setAttribute('src', `${obj.authorPhoto}`);
+    // imgDiv.setAttribute('alt', 'authors picture');
+   
+    authNameSpan.textContent = `By ${obj.authorName}`;
+    
+    // return
+    return parentDiv;
+    
+    } //end lambCards comp
+
+    //loop through our api to find each structure
+    articleAPI.then(response => {
+        //js structure
```

Comment:
You could have stored the entire response.data.articles into 1 variable, used something like Object.keys to transform the JSON to an array and done a forEach on the entire array that way as well.